packages/plugins/rubygem-radcli/rubygem-radcli.spec

```
@@ -1,87 +1,103 @@
+# Generated from radcli-1.0.0.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name radcli
 
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 1.0.0
-Release: 1%{?dist}
+Release: 2%{?dist}
 Summary: A Ruby interface for the adcli library
 Group: Development/Languages
 License: Artistic-2.0
 URL: https://github.com/martencassel/radcli
-Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
-Requires: ruby(release)
-Requires: ruby
-Requires: ruby(rubygems)
-BuildRequires: ruby(release)
-BuildRequires: ruby-devel
-BuildRequires: rubygems-devel
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+BuildRequires: %{?scl_prefix}rubygem(rake-compiler)
 BuildRequires: krb5-devel
 BuildRequires: openldap-devel
 BuildRequires: cyrus-sasl-devel
-Provides: rubygem(%{gem_name}) = %{version}
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby-devel
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
 
 %description
 The radcli library provides a Ruby interface for performing actions on
 a Active Directory domain using the realmd/adcli tool library.
 
 
 %package doc
-Summary: Documentation for %{name}
+Summary: Documentation for %{pkg_name}
 Group: Documentation
-Requires: %{name} = %{version}-%{release}
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
 BuildArch: noarch
 
 %description doc
-Documentation for %{name}.
+Documentation for %{pkg_name}.
 
 %prep
+%{?scl:scl enable %{scl} - << \EOF}
 gem unpack %{SOURCE0}
+%{?scl:EOF}
 
 %setup -q -D -T -n  %{gem_name}-%{version}
 
+%{?scl:scl enable %{scl} - << \EOF}
 gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
 
 %build
 # Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
 gem build %{gem_name}.gemspec
+%{?scl:EOF}
 
 # %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
 # by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
 %gem_install
+%{?scl:EOF}
 
 %install
 mkdir -p %{buildroot}%{gem_dir}
-cp -a .%{gem_dir}/* \
+cp -pa .%{gem_dir}/* \
```

We've simplified the tempate in f02a98ade2c230f3968cd12e0ac220262f10109c:
```suggestion
cp -a .%{gem_dir}/* \
```